qiskit/tools/visualization/_circuit_visualization.py

```diff
@@ -63,7 +65,7 @@ def circuit_drawer(circuit,
                    scale=0.7,
                    filename=None,
                    style=None,
-                   output=None,
+                   output="text",
```

Comment:
I don't think we can do this yet. While the eventual goal is to switch this to be the default we have to allow for a transition period for existing users. I would update the docstrings (and changelog too probably) to say ascii will be the default after we deprecate the current default.

Reply:
Changed back in bfba283.

If `circuit_drawer` was introduced post 0.6 (correct me if I'm wrong here), do we really need a deprecate message if its interface?

Reply:
Well that's where it's a bit confusing. `circuit_drawer` has actually existed since well before 0.6.0, we only recently added the `output` kwarg and the other new options in an attempt to make it a single method for drawing circuits in the future. (as opposed to having `circuit_drawer` and also `plot_circuit`, `latex_circuit_drawer`, and `matplotlib_circuit_drawer` which all did the same thing) But, we have to be careful not to break existing users of circuit_drawer() by changing all the default behavior of the function. So in #1055 we deprecated all the other methods and added the new options and deprecated the old default behavior of `circuit_drawer`.